core/src/main/java/feign/Util.java

```diff
@@ -139,6 +139,18 @@ public static String emptyToNull(String string) {
     return collection.toArray(array);
   }
 
+  public static boolean isHttpUrl(CharSequence value) {
```

Comment:
as these methods are only used once, move them to RequestTemplate.java and mark private.